Bottom: 6d5c93a57ea656fe17f1c2bb4991db8186c9f828
Top:    f5f2177f7b75adf5b3e3e07accc61ca413551886
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-01-28 22:06:14 +0000

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!






---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 2dcf4b8..5e56536 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -312,13 +312,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
